src/modules/projects/components/ProjectReferralPostingForm.tsx

```diff
@@ -34,10 +35,28 @@ export const ProjectReferralPostingForm: React.FC<Props> = ({
     },
     onError: (apolloError) => setErrors({ ...emptyErrorState, apolloError }),
   });
+
+  const definition = useMemo(() => {
+    // Hacky way to drop the "Assigned" option if this posting is in AcceptedPending status.
+    // This lets user change from AcceptedPending=>DeniedPending which is sometimes necessary.
+    const definitionCopy = { ...ReferralPostingDefinition };
```

Comment:
Because this is a shallow copy, changing `item[0].pickListOptions` will change the original object as well. This can result in some really weird bugs. Recommend we change this.

For small objects that can be serialized, probably the easiest way to get a deep copy is:
```suggestion
    const definitionCopy = JSON.parse(JSON.stringify(ReferralPostingDefinition)) as typeof ReferralPostingDefinition;
```

Reply:
Ah thank you, good catch